compiam/rhythm/meter/akshara_pulse_tracker/models.py

```diff
@@ -518,6 +527,16 @@ def extract(self, file_path, verbose=True):
         onsTs = onsTs[offsetIndex:]
         sectEnd = np.append(sectEnd, onsTs[-1])
 
+        # Get sections
+        sections = {"startTime": 0, "endTime": 0, "label": ""}
+        sections["startTime"] = np.round(sectStart, params.roundOffLen)
+        sections["endTime"] = np.round(sectEnd, params.roundOffLen)
+        labelStr = ("Alapana", "Kriti")
+        if sectEnd.size == 2:
```

Comment:
this is basically so that it includes all outputs of the original Rhythm Extractor that was developed and used to compute the features in Dunya. In that sense, we will be able to compute the features that are available in the Dunya database through this extractor in Compiam. Probably we should add documentation about that.

Reply:
OK. We should ensure that the other places that use this doesn't fail. Like the web book notebooks for example!

Reply:
ooops that's good point